guide.md

```diff
@@ -71,22 +71,83 @@ Do not confuse burnchain modes **neon** and **helium** with run loops **Neon** a
 ## What is testnet?
 
 ## What is sortition?
+Sortition is the process of selecting a miner principal to broadcast block data
+to the stacks nodes for the next winning block. Before candidating, miners must
+commit the chain tip they're building on and the hash of the block they intend
+to broadcast if they win. The details of the process looks slightly different
+in *proof of burn* (PoB) and *proof of transfer* (PoX).
 
-## What are tenures?
+**In *proof of burn***, miners candidate to have their principals be the leader
+of an epoch by burning tokens on the burn chain, e.g. Bitcoin. The
+leader is then selected through a *verifiable random function* (VRF). The VRF
+has the following properties
+
+- The likelihood of a principal to be elected is proportional to the amount of
+  tokens the principal has burned.
+- The output of the VRF cannot be predicted before the burn transaction has been
+  included in a block.
+
+***Proof of transfer*** extends *Proof of burn* by instead of burning tokens to candidate,
+miners transfer tokens to addresses of STX holders who participate in stacking. The
+remainder of the sortition process remains the same as in PoB.
+
+Sortition is defined in SIP-001
+[here](https://github.com/stacksgov/sips/blob/main/sips/sip-001/sip-001-burn-election.md#step-3-sortition).
+The PoX adaptation is defined in [SIP-007](https://github.com/stacksgov/sips/blob/main/sips/sip-007/sip-007-stacking-consensus.md).
+
+## What is a tenure?
+A tenure is the period during which an elected leader propagates transaction data.
+The tenure is terminated when a new burn chain block arrives.
+
+For more details, see the tenure implementation in the stacks blockchain
+[here](https://github.com/stacks-network/stacks-blockchain/blob/master/testnet/stacks-node/src/tenure.rs#L26).
 
 ## What are epochs?
+Epochs in the Stacks blockchain correspond to the leader and leader candidate state
```

Comment:
Can we add a link to epoch enum definition in the code?

Reply:
Also, I'm confused about this definition for epoch. My understanding is that an epoch is a period in the blockchain's history with defined consensus rules. New epochs are created to accommodate changes in these rules.

Reply:
I will link the definition. Yeah I get the confusion, this description is how epochs are defined in SIP-001 (https://github.com/stacksgov/sips/blob/main/sips/sip-001/sip-001-burn-election.md#definitions). However, consensus-breaking changes (Epoch 2.1, Epoch 3) also reuse the term. I'll update the text to disambiguate the two variants.